src/virtualenv/activation/__init__.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import absolute_import, unicode_literals
+
+from .bash import BashActivator
```

Comment:
If this imports all the objects anyhow maybe the modules they come from should be private?

Reply:
good point, I'll have a think about it, as part of the pre-release cleanup 👍